meta.yml

```
@@ -37,8 +37,8 @@ opam-file-maintainer: vovaglad00@gmail.com
 
 license:
   fullname: GNU General Public License v2.1
```

```suggestion
  fullname: GNU Lesser General Public License v2.1
```
And the rest should be regenerated